apps/weshipit/pages/podcast/[slug].tsx

```diff
@@ -0,0 +1,176 @@
+import { useRouter } from 'next/router';
+import { Hyperlink, Button, SpotifyIcon, ApplePodcastIcon } from '@weshipit/ui';
+import { Layout } from '../../components/layout';
+import { podcastEpisodes } from '../../fixtures/podcast-episodes.fixture';
+import { useEffect, useState } from 'react';
+import { Transcript } from '../../components/transcript';
+import { parseTranscript, TranscriptEntryType } from '../../utils/transcript';
+import { PodcastNavigation } from '../../components/podcast-navigation';
+
+export default function PodcastEpisodePage() {
+  const router = useRouter();
+  const { slug } = router.query;
+
+  const episode = podcastEpisodes.find((ep) => ep.slug === slug);
+
+  const [transcriptEntries, setTranscriptEntries] = useState<
+    TranscriptEntryType[] | null
+  >(null);
+
+  useEffect(() => {
```

Comment:
do not add a useEffect 
SPA are cr4p for SEO

use the same rendering strategy as Gatsby 
https://nextjs.org/docs/pages/building-your-application/rendering/static-site-generation

and we should have another page for transcript 

I think it's possible to have `apps/weshipit/pages/podcast/[slug]-transcript.tsx`

see #238